iaso/tests/api/test_tasks.py

```diff
@@ -4,32 +4,34 @@
 from django.utils import timezone
 
 from iaso import models as m
-from iaso.models.base import ERRORED, QUEUED
-from iaso.api.tasks import TaskSerializer
+from iaso.models.base import ERRORED, QUEUED, RUNNING, EXPORTED, SUCCESS
+from iaso.api.tasks.serializers import TaskSerializer
 from iaso.test import APITestCase, TestCase
 
 
-DT = datetime.datetime(2023, 12, 20, 15, 0, 0, 0, tzinfo=timezone.utc)
+CREATED_AT = datetime.datetime(2023, 12, 20, 15, 0, 0, 0, tzinfo=timezone.utc)
 
 
-@time_machine.travel(DT, tick=False)
+@time_machine.travel(CREATED_AT, tick=False)
 class TaskSerializerSerializerTestCase(TestCase):
     """
     Test Task serializer.
     """
 
+    maxDiff = None
```

Comment:
TBH I'm always tempted to put that in the base class…

Reply:
Agreed, I'll do it